types/react-phone-number-input/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for react-phone-number-input 3.0
+// Project: https://github.com/AdrienEtienne/react-phone-number-input
+// Definitions by: Peter Yang <https://github.com/PeterYangIO>
+//                  Adrien Etienne <https://github.com/AdrienEtienne>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.0
+
+import * as React from "react";
+
+export type NumberFormat = "NATIONAL" | "National" | "INTERNATIONAL" | "International";
+
+export function formatPhoneNumber(value: string, format?: NumberFormat): string;
+export function formatPhoneNumberIntl(value?: string): string;
+export function isValidPhoneNumber(value?: string): boolean;
+
+export interface FlagsMap { [countryCode: string]: React.Component<object, object>; }
+
+export interface CountrySelectComponentProps {
+    className?: string;
+    disabled?: boolean;
+    name?: string;
+    onBlur?: () => void;
+    onChange?: (value: string) => void;
+    onFocus?: () => void;
+    options?: Array<{ value?: string, label: string, icon: React.Component }>;
+    tabIndex?: (number | string);
+    value?: string;
+}
+
+export interface InputComponentProps {
+    // Required props
+    onChange: (value: string) => void;
+    value: string;
+    // Optional props
+    onFocus?: () => void;
+    onBlur?: () => void;
+    country?: string;
+    metadata?: object;
+}
+
+export interface PhoneInputProps {
```

Comment:
from the docs "See the list of all available props for <PhoneInput/>. All properties not listed there will be passed through to the phone number <input/> component." 
The props for this should also take any valid `<input/>` props.

```typescript
export interface PhoneInputProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "onChange" | "id"> {
```

Reply:
Thanks.
Have created a PR for that: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41898